core/XH.js

```diff
@@ -468,9 +468,12 @@ class XHClass {
             try {
                 await this.installServicesAsync(EnvironmentService);
             } catch (e) {
+                const loc = window.location,
+                    pingURL = XH.isDevelopmentMode ? `${XH.baseUrl}ping` : `${loc.protocol}//${loc.host}/ping`;
```

Comment:
Will still need to work in baseUrl for the production case - it will be /api/ but that's important as otherwise the link won't pass through to the back-end server.

Believe we can use `${location.origin}${XH.baseUrl}ping` - that looks correct on deployed Toolbox.